test/resource_pci_cluster_test.go

```diff
@@ -51,13 +51,14 @@ func TestAccPCIClusterResize(t *testing.T) {
 	validConfig, _ := ioutil.ReadFile("data/valid_with_resizable_pci_cluster.tf")
 	username := os.Getenv("IC_USERNAME")
 	apiKey := os.Getenv("IC_API_KEY")
-	oriConfig := fmt.Sprintf(string(validConfig), username, apiKey)
+	hostname := getOptionalEnv("IC_API_URL", instaclustr.DefaultApiHostname)
+	oriConfig := fmt.Sprintf(string(validConfig), username, apiKey, hostname)
```

Comment:
[Question] - what does `oriConfig` mean/stand for?

Reply:
Truthfully i don't know - i would assume original config but i will clarify and update

Reply:
Haha yeah I think that makes sense, also just realized you didn't introduce this variable, sorry for the comment! 😅